### ISSUE TITLE
Mutator dialog is displayed in the wrong location / size momentarily

### ISSUE THREAD

**Issue by samelhusseini** [google/blockly#3170]
**Describe the bug**
When first opening the mutator dialog, and on (some) subsequent opens, the dialog is displayed elsewhere on the screen and then resized. 

**To Reproduce**

1. Go to playground
2. Drag out any block with a mutator dialog
3. Open the dialog, repeatedly
4. Notice that sometimes the dialog will show elsewhere on the screen, and then resize to its correct location.

**Expected behavior**

Size and position correctly when opened.

**Screenshots**

![mutator_size_flash](https://user-images.githubusercontent.com/16690124/66245886-81ac2880-e6c5-11e9-83d6-d6595b5271f5.gif)

I can repro on Chrome and Safari.

**Additional context**

**Comment by samelhusseini** [comment]
I've traced this back to the removal of this line: https://github.com/google/blockly/commit/9e5df6216a28a21ea8c8f423afe1bc50195ab5d7#diff-d4e069a460b289628cfaed6e6649b43aL300

@BeksOmega any context as to why this line was removed?

**Comment by BeksOmega** [comment]
> I've traced this back to the removal of this line: [9e5df62#diff-d4e069a460b289628cfaed6e6649b43aL300](https://github.com/google/blockly/commit/9e5df6216a28a21ea8c8f423afe1bc50195ab5d7#diff-d4e069a460b289628cfaed6e6649b43aL300)
> 
> @BeksOmega any context as to why this line was removed?

I think I removed it originally because the JSDoc is wrong. I was probably thinking something like "why are we calling a resize listener when the bubble hasn't resized?"

But looking at it, adding that call back in is the wrong solution as well. It sets the bubbles size and causes a reflow when it would be more efficient to just give the bubble the correct information in the constructor.

I've got about half a solution atm. If I don't get it finished tonight I should have something up tomorrow.
